src/brod_group_coordinator.erl

```diff
@@ -49,6 +49,7 @@
 
 %% default configs
 -define(SESSION_TIMEOUT_SECONDS, 30).
+-define(REBALANCE_TIMEOUT_SECONDS, 2).
```

Comment:
To make it backward compatible, please default to `SESSION_TIMEOUT_SECONDS` here.
If I remember correctly, this is how newer version Kafka handles version `0` join request:
it takes the session timeout for rebalance timeout.

Reply:
Thanks, will do :+1: